README.md

```diff
@@ -16,13 +16,49 @@ until it's fully presented or fully dismissed. It's *not* (yet) a complete imple
 the Maps app simply because our specific needs dictated something else. We intend to continue working on it to address
 that limitation.
 
+Please do play with the demo app and try different configuration options because there are so many ways to configure
+`DrawerKit` that the gif below is at most a pathetic representation of everything that the library can do.
+
 <p align="center">
-	<a href="https://github.com/Babylonpartners/DrawerKit/">
-	    <img src="drawers.gif" alt="DrawerKit" />
-	</a>
+	<table>
+		<tr>
+			<td>
+				<a href="https://github.com/Babylonpartners/DrawerKit/">
+					<img src="drawers2.gif" alt="DrawerKit" width="288" height="514" />
+				</a>
+			</td>
+			<td>
+				<a href="https://github.com/Babylonpartners/DrawerKit/">
+					<img src="drawers3.gif" alt="DrawerKit" width="288" height="514" />
+				</a>
+			</td>
+		</tr>
+	</table>
 </p>
 
-## What's new since v. 0.1.1?
+## What's new in version 0.3.0?
+
+Please note that v. `0.3.0` is not backward-compatible with v. 0.2.
```

Comment:
`backwards-compatible`

Reply:
Ah, yes. Thanks!

Reply:
Fixed in https://github.com/Babylonpartners/DrawerKit/pull/30/commits/5cb629b36630fe49d57262310ea2523c7e4a65b9.